src/resolve/resolved-dns-packet.h

```diff
@@ -117,11 +117,14 @@ static inline uint16_t DNS_PACKET_RCODE(DnsPacket *p) {
 
 static inline uint16_t DNS_PACKET_PAYLOAD_SIZE_MAX(DnsPacket *p) {
 
-        /* Returns the advertised maximum datagram size for replies, or the DNS default if there's nothing defined. */
+        /* Returns the advertised maximum size for replies, or the DNS default if there's nothing defined. */
 
         if (p->opt)
                 return MAX(DNS_PACKET_UNICAST_SIZE_MAX, p->opt->key->class);
```

Comment:
Hmm, this part seems completely bogus. `class` is either 0x01 or 0xFF, but it is not a meaningful size. I guess some translation from class to size was intended here. @poettering ?

Reply:
Ah, it's the OPT RR "class". So yeah, this seems to be correct.